src/components/Menu/styles/withButtons/StyledItemLink.js

```diff
@@ -0,0 +1,35 @@
+import styled from 'styled-components'
+import Link from 'components/Link'
+
+const ItemLink = styled(Link)`
+  height: 100%;
+  width: 100%;
+
+  padding: 0 ${({ theme }) => theme.components.menu.withButtons.link.spaces.paddingWidth};
```

Comment:
You should deconstruct `theme.components.menu.withButtons.link` because it's way too long and you use it multiple times

Reply:
See #56